### PR TITLE
Add settings to ImageTextBlock to fill viewport in mobile

### PR DESCRIPTION
**Pull request by gustaveen**
## Justify why they are needed
Updated design for A/B test
image

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2513** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2513" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2513?utm_source=stack-comment).

File: apps/store/src/blocks/ImageTextBlock.tsx [hunk 1]
```diff
@@ -48,6 +49,7 @@ export const ImageTextBlock = ({ blok, nested }: ImageTextBlockProps) => {
           textHorizontalAlignment={blok.textHorizontalAlignment}
           imagePlacement={blok.imagePlacement}
           mediaPlacementMobile={blok.mediaPlacementMobile}
+          fillViewport={blok.fillViewport}
```
Comment (robinandeer) [inline apps/store/src/blocks/ImageTextBlock.tsx:52]: nit: this is only for mobile, yes? Should we make that clear in the prop name?

**Review by robinandeer — APPROVED** [review]
Looks good :D

Added via Giphy

File: apps/store/src/blocks/ImageTextBlock.tsx (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/blocks/ImageTextBlock.tsx]: Yes, let's!

**Pull request merged**